documentation-site/pages/components/input.mdx

```diff
@@ -23,16 +23,17 @@ import Clearable from 'examples/input/clearable.js';
 import Password from 'examples/input/password.js';
 import BasicUncontrolled from 'examples/input/basic-uncontrolled.js';
 
-import Overrides from '../../components/overrides';
-import Yard from '../../components/yard/yard';
 import {StatefulInput} from 'baseui/input';
 import * as InputExports from 'baseui/input';
 
+import Yard from '../../components/yard/index';
+import inputYardConfig from '../../components/yard/config/input';
+
 export default Layout;
 
 # Input
 
-<Yard componentName="Input" />
+<Yard componentName="Input" defaultHeight={726} {...inputYardConfig} />
```

Comment:
is there a default 'defaultHeight'? 😆I would be nice to not specify this prop if it looks fine by default, but maybe the TS error will be enough

Reply:
Well, this is used when the yard content is not loaded yet, it preallocates the vertical space and prevents the page to jump when loaded.